tests/test_main.py

```diff
@@ -0,0 +1,25 @@
+from typer.testing import CliRunner
+
+runner = CliRunner()
+
+
+def test_gatorgrade_runs_correct():
+    """Test that ensures that the default command runs correctly."""
+    assert True
+
+
+def test_generate_creates_correctly():
```

Comment:
In the same vein as my comment above, this should be named something like `test_generate_runs` or (if it is indeed checking the contents of the generated yml file) `test_generate_creates_valid_yml`. However, I would suggest that your CLI test suite *only* tests the plumbing; i.e. don't test any of the generated content or behavior, and instead use something like [pytest-mock](https://pypi.org/project/pytest-mock/) to test that the behavior functions written by other teams are called as intended.